.gitignore

```diff
@@ -3,11 +3,6 @@ node_modules
 npm-debug.log
 .npm
 
-# OSX system files, the bane of our existence
-.DS_Store
-.AppleDouble
-.LSOverride
```

Comment:
are we expecting users to understand to ignore these at the user/global level? hoping it doesn't come back to cause trouble

Reply:
Right, I feel that we should avoid having env specific / platform specific ignores. And my experience in general is that these hardly ever get committed. And if ever, they can just delete them.